lib/createExplorer.js

```diff
@@ -74,11 +71,9 @@ module.exports = function (options) {
       })
       .then((result) => {
         if (result) return result;
-
-        const nextDirectory = joinPath(splitPath(directory).slice(0, -1));
-        if (!nextDirectory || directory === options.stopDir) return null;
-
-        return searchDirectory(nextDirectory);
+        if (!splitSearchPath) splitSearchPath = splitPath(directory);
+        if (!splitSearchPath.pop() || directory === options.stopDir) return null;
+        return searchDirectory(joinPath(splitSearchPath), splitSearchPath);
```

Comment:
I'm not sure what this means ... Are you saying that `path.resolve()` is not working when you access a file in a drive different from where `process.cwd()` is